data/Sri Dasam Granth/0007.json

```diff
@@ -792,7 +792,7 @@
         "id": "5MTE",
         "source_page": 8,
         "source_line": 3,
-        "gurmukhi": "suinq hYN ]",
+        "gurmukhi": "su inq hYN ]",
```

Comment:
This one and the line above it are both missing Adhaks

Reply:
Line above meaning ID 8ARS

Reply:
TPJA and K14Q

Perhaps many others are missing too or perhaps this is by design and I'm unknowing

Reply:
require notes on all removed adhaks that it is a deviance from SGPC and reasoning why

Reply:
As discussed in Slack, adhaks will be added into pronunciation and notes